kuwala/core/canvas/src/utils/DataBlockUtils.js

```diff
@@ -0,0 +1,46 @@
+import DataBlockDTO from "../data/dto/DataBlockDTO"
+import {getDatabaseTitleValue} from "./SchemaUtils";
+
+export function getDataBlockAddressString ({dataBlockResponse, dataSource, schema}) {
+
+    function lookUpTableCategories (schema_name, table_name) {
+        let categoryName = 'tables';
+        const targetSchema = schema.find((el) => el.schema === schema_name);
+        for (const cat of targetSchema.categories) {
+            if (!cat.tables) break;
+            if (cat.tables.includes(table_name)) {
+                categoryName = cat.category;
+            }
```

Comment:
```{
    "schema": "dbt_kuwala",
    "categories": [
      {
        "category": "tables",
        "tables": [
          "city_not_null_3lcarv_fpm9ja",
          "is_true_3vbvuy_fpm9ja",
          "is_truee_kuqvtg_fpm9ja",
          "remove_null_4cakto_yhqxtb",
          "remove_null_city_4ajykt_fpm9ja"
        ]
      },
      {
        "category": "views",
        "tables": [
          "facebook_ads_3vftae",
          "fb_ads_fnykdz",
          "mock_data_yhqxtb",
          "postgres_ads_3nxv2e",
          "postgres_new_fpm9ja",
          "postgres_yhqxtb",
          "stg_public_facebook_ads",
          "stg_public_mock_data_3"
        ]
      }
    ]
  },```
  
i already did, since it loops the array value of `targetSchema.categories` it will return the category name when the target table is found on the tables attribute